### PR TITLE
Fix images paths sorry

### PR DESCRIPTION
**Pull request by khallebal**
I didn't notice before the merge

### CODE REVIEW

File: content/guides/booting.md [hunk 1]
```diff
@@ -99,3 +99,31 @@ Adding Haiku to your [GRUB Legacy](http://www.gnu.org/software/grub/grub-legacy.
...
     chainloader +1
+
+### Adding Haiku to the Windows bootloader using EasyBCD
+
+* First get EasyBCD from [this site](https://neosmart.net/EasyBCD/)
+* Install it as you would install any windows software
+* When you launch it you'll see 2 panels
+* On the left panel click on "Add new entry"
+
+[](/files/guides/easybcd_0.png)
```
Comment (humdingerb) [inline content/guides/booting.md]: Shouldn't this (and following) be: ``
You could also add an actual title in the `[]` and the images have an all-capital .PNG suffix.

**Comment by khallebal** [comment]
I did not use capitals because i noticed the other guides used lower-case suffixes, maybe there is a reason?
Anyway, i've changed it now.

**Comment by humdingerb** [comment]
> I did not use capitals because i noticed the other guides used lower-case suffixes, maybe there is a reason?
> Anyway, i've changed it now.

Well, the new screenshots all had capital ".PNG". You could've changed those to ".png", but it either way is OK. Merging.

**Comment by humdingerb** [comment]
Or not... See "Conflicting files  `content/guides/booting.md`"...

**Comment by khallebal** [comment]
Conflict fixed, hopefully it's all fine now.

**Comment by humdingerb** [comment]
Thanks! How sure are we the spaces in the image names are working without replacing with %20 in the links?

**Comment by khallebal** [comment]
I have no idea? this is all new to me, do you have an idea?

**Comment by humdingerb** [comment]
To avoid possible issues and the aesthetics of paths peppered with "%20", I suggest to just change the image filenames and links to camel case, e.g. "AddNewEntry.png".

**Comment by khallebal** [comment]
I think it's required after all, i just noticed that the paths get highlighted in my editor after filling the spaces with %20.
Pushing the changes now.

**Comment by khallebal** [comment]
Done.

**Comment by humdingerb** [comment]
Thank you!

**Pull request merged**